nomad/plan_normalization_test.go

```diff
@@ -62,5 +62,5 @@ func TestPlanNormalize(t *testing.T) {
 	}
 
 	optimizedLogSize := buf.Len()
-	assert.True(t, float64(optimizedLogSize)/float64(unoptimizedLogSize) < 0.62)
+	assert.Less(t, float64(optimizedLogSize)/float64(unoptimizedLogSize), 0.63)
```

Comment:
The compression value needed to be raised to account the new MemoryMaxMB field. Seems like a pretty odd test that will effectively fail anytime we add fields to allocs.  It's nice to keep track of the value overtime, but don't know of a better way to track it, so just changed the value here.